doc/varnish-configuration.rst

```diff
@@ -202,7 +202,10 @@ User Context
 
 Feature: :doc:`user context hashing <user-context>`
 
-Subroutines are provided in ``resources/config/varnish-[version]/fos_user_context.vcl``.
+Subroutines are provided in ``resources/config/varnish-[version]/fos_user_context.vcl`` and ``resources/config/varnish-[version]/fos_user_context_url.vcl``. The default lookup URL is `/_fos_user_context_hash`. You 
+may change it by implementing your own ``user_context_hash_url`` subroutine. 
```

Comment:
lets rephrase this to say that fos_user_context.vcl needs the user_context_hash_url subroutine that sets the url to the request lookup url. The default URL is /_fos_user_context_hash and you can simply include ..._url.vcl in your configuration to provide this. If you need a different URL, include a custom file.